app/api/routers/breeze_buddy/auth/__init__.py

```
@@ -209,3 +215,32 @@ async def logout():
     response.headers["Pragma"] = "no-cache"
     response.headers["Expires"] = "0"
     return response
+
+
+@router.post("/auth/launch-code", response_model=LaunchCodeResponse)
+async def mint_launch_code_route(
+    request: LaunchCodeRequest,
+    current_user: UserInfo = Depends(get_current_user_with_rbac),
```

Ruff (0.15.18)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)